### DISCUSSION TITLE
Love this tool so much and also -- any plans to integrate with Obsidian?

### DISCUSSION THREAD

**Discussion by clening** [jzillmann/pdf-to-markdown#64] [Ideas]
I love pdf2markdown so much, and very much appreciate that it's an online tool (I haven't figured out how to make the code run in linux, so for now I am content with the website). 

I was wondering though -- was there any plans to integrate PDF to Markdown in a PKM (personal knowledge management) tool like Obsidian? I have to convert a boatload of PDFs to markdown to put them in Obsidian and from a purely self-serving perspective, it would vastly improve my efficiency if such a conversion tool existed within the app. 

I'm sure you have enough on your plate, but I thought I'd at least ask. Also, is there any way to provide a little financial support for all your work? I relied on pdf2markdown a great deal for a recent project, and will continue to sing its praises and use it in the future. I even wrote a little article on what I was working on and how helpful pdf2markdown was to achieving my goals. https://careylening.substack.com/p/the-power-of-links-and-second-brains-d1d

**Comment by clening** [comment]
Moved this to a FR in Issues. Closing in here.

**Reply by jzillmann** [reply]
Responded in https://github.com/jzillmann/pdf-to-markdown/discussions/64
